testsuite/openshift/kuadrant.py

```diff
@@ -0,0 +1,27 @@
+"""Kuadrant CR object"""
+
+from openshift_client import selector
+
+from testsuite.openshift import OpenShiftObject, modify
+
+
+class KuadrantCR(OpenShiftObject):
+    """Represents Kuadrant CR objects"""
+
+    LIMITADOR = "limitador-limitador"
+
+    @modify
+    def set_limitador_options(self, replicas: dict):
+        """Configure Limitador spec"""
+        self.model.spec.setdefault("limitador", {})
+        self.model.spec["limitador"] = replicas
+
+    def delete(self, ignore_not_found=True, cmd_args=None):
```

Comment:
Not sure if this is needed, we do not need to protect against stupidity of the test developer